lib/erlef/captcha.ex

```diff
@@ -1,4 +1,9 @@
 defmodule Erlef.Captcha do
+  @moduledoc """
+  Erlef.Captcha - Captcha related verification functions
+  """
+  @recaptch_url Application.get_env(:erlef, :recaptcha_url)
```

Comment:
that ones creates a risk if you take a test build and deploy to prod, where it starts using the test URL for prod capchas right?

Reply:
Technically, this should never happen. The different url's are defined in config. Maybe hardcoding it here and doing some if/else work might be safer in regards to the behavior being right there. We could also fetch it at runtime vs baking it in, but I think that still suffers from the potential to accidentally have the wrong env set, etc.

Reply:
Yeah I mean the tricky issue is that the beam artifact has the configuration value baked from the build, so the binaries aren't portable across envs, but that's a minor thing.